### PR TITLE
refactor: simplified index.tsx

### PR DESCRIPTION
**Pull request by Simplyalex99**
Separated the contents in  web/src/pages/index.tsx  into seperate components in ui/src/components/home/sections.
Currently, the files can not be imported using "@oasis-sh/ui" or even be imported since the dist folder does not contain it  resulting in build error.

_NOTE:_ Was facing issues with setting this up using yarn build UI command. I tried to remove the  dist folder in ui and runned the yarn commands which did work creating the files in ui/dist folder but this also prevents some other important thing from downloading resulting in weird styling problems and components problems. 

**Please use with caution.** I tested the refractor code and it work. However, again the only issue is setting it up in UI folder

### CODE REVIEW

File: packages/ui/src/components/home/sections/PostsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+import {
+    CreatePostInput
+} from '../CreatePostInput';
+import { Post } from "../../post/Post";
+
+import StyledMarkdown from '../../../../../web/src/markdown/StyledMarkdown';
```
Comment (heybereket) [inline packages/ui/src/components/home/sections/PostsSection.tsx:8]: Don't import things like this from other package files, add it as props instead.

File: packages/web/src/pages/index.tsx [hunk 2]
```diff
@@ -58,121 +60,17 @@ const HomePage: React.FC<IndexPageProps> = ({ vars }) => {
...
-              </Sidebar>
-              <Sidebar title="Find New People" className="mt-10">
-                <div className="mt-6 space-y-3">
-                  <FollowUser name="Bereket" username="heybereket" />
-                  <FollowUser name="Alex" username="alexover1" />
-                  <FollowUser name="Sam" username="samjakob" />
-                </div>
-              </Sidebar>
+
+              <SidebarSection></SidebarSection>
```
Comment (heybereket) [inline packages/web/src/pages/index.tsx:73]: Don't combine both of the Sidebar's into 1, keep them how it originally was.

File: packages/web/src/pages/index.tsx [hunk 3]
```diff
@@ -58,121 +60,17 @@ const HomePage: React.FC<IndexPageProps> = ({ vars }) => {
...
-                    <a className="flex items-center space-x-0.5 mt-2">
-                      <p className="font-bold text-lg text-primary">
-                        View Profile
-                      </p>
-                      <RightArrow className="text-primary" />
-                    </a>
-                  </Link>
-                </>
-              )}
+              <UserSection currentUserLoading={currentUserLoading} user={user}></UserSection>
```
Comment (heybereket) [inline packages/web/src/pages/index.tsx:63]: Rename this to "UserProfileCard" or "ProfileSection".

File: packages/web/src/pages/index.tsx [hunk 4]
```diff
@@ -12,6 +12,8 @@ import {
   Post,
   FollowUser,
   CreatePostInput,
+  UserSection,
+  PostsSection, SidebarSection, FriendActivitySection
```
Comment (heybereket) [inline packages/web/src/pages/index.tsx:16]: Keep imports in separate lines, don't import them in one whole line like this.

File: packages/ui/src/index.ts [hunk 5]
```diff
@@ -13,7 +13,10 @@ export { FollowUser } from './components/home/FollowUser';
 export { FriendActivity } from './components/home/FriendActivity';
 export { Sidebar } from './components/home/Sidebar';
 export { CreatePostInput } from './components/home/CreatePostInput';
-
+export {UserSection} from "./components/home/sections/UserSection";
+export {FriendActivitySection} from "./components/home/sections/FriendActivitySection";
+export {PostsSection} from "./components/home/sections/PostsSection";
+export {SidebarSection} from "./components/home/sections/SidebarSection";
```
Comment (heybereket) [inline packages/ui/src/index.ts:19]: Add spaces in these imports: (example: export { Button } from 'path')

File: packages/ui/src/components/home/sections/SidebarSection.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            <Sidebar title="Find New People" className="mt-10">
+                <div className="mt-6 space-y-3">
+                    <FollowUser name="Bereket" username="heybereket" />
+                    <FollowUser name="Alex" username="alexover1" />
+                    <FollowUser name="Sam" username="samjakob" />
+                </div>
+            </Sidebar>
+        </>
+    )
+}
```
Comment (heybereket) [inline packages/ui/src/components/home/sections/SidebarSection.tsx:34]: Revert this file, as mentioned earlier. They are separate sidebars.

**Review by heybereket — CHANGES_REQUESTED** [review]
Nice work! Left some feedback:

**Review by alexover1 — CHANGES_REQUESTED** [review]
Do everything @heybereket said, also are you using prettier? It should automatically fix a lot of those format issues. Also, fix the merge conflicts.

**Comment by Simplyalex99** [comment]
Okay, I will make the changes tonight or tomorrow.

**Pull request closed without merging**